src/language-css/printer-postcss.js

```diff
@@ -39,19 +39,30 @@ const isSCSSControlDirectiveNode = utils.isSCSSControlDirectiveNode;
 const isDetachedRulesetDeclarationNode = utils.isDetachedRulesetDeclarationNode;
 const isRelationalOperatorNode = utils.isRelationalOperatorNode;
 const isEqualityOperatorNode = utils.isEqualityOperatorNode;
+const isMultiplicationNode = utils.isMultiplicationNode;
+const isDivisionNode = utils.isDivisionNode;
+const isAdditionNode = utils.isAdditionNode;
 const isMathOperatorNode = utils.isMathOperatorNode;
 const isEachKeywordNode = utils.isEachKeywordNode;
 const isParenGroupNode = utils.isParenGroupNode;
 const isForKeywordNode = utils.isForKeywordNode;
 const isURLFunctionNode = utils.isURLFunctionNode;
 const isIfElseKeywordNode = utils.isIfElseKeywordNode;
-const hasLessExtendValueNode = utils.hasLessExtendValueNode;
-const hasComposesValueNode = utils.hasComposesValueNode;
-const hasParensAroundValueNode = utils.hasParensAroundValueNode;
+const hasLessExtendNode = utils.hasLessExtendNode;
+const hasComposesNode = utils.hasComposesNode;
+const hasParensAroundNode = utils.hasParensAroundNode;
+const hasEmptyRawBefore = utils.hasEmptyRawBefore;
 const isKeyValuePairNode = utils.isKeyValuePairNode;
 const isDetachedRulesetCallNode = utils.isDetachedRulesetCallNode;
 const isPostcssSimpleVarNode = utils.isPostcssSimpleVarNode;
 const isSCSSMapItemNode = utils.isSCSSMapItemNode;
+const isInlineValueCommentNode = utils.isInlineValueCommentNode;
+const isHashNode = utils.isHashNode;
+const isLeftCurlyBraceNode = utils.isLeftCurlyBraceNode;
+const isRightCurlyBraceNode = utils.isRightCurlyBraceNode;
+const isWordNode = utils.isWordNode;
+const isColonNode = utils.isColonNode;
+const isMediaAndSupportsKeywords = utils.isMediaAndSupportsKeywords;
```

Comment:
What do you think about (maybe in a future PR) getting rid of all of these declarations and just prefixing the variables with `utils.` when they’re used?

Reply:
@j-f1 maybe, i think we can import `utils` as `u` for less breaking lines (example in `if`)